app/scripts/filters/resources.js

```diff
@@ -1084,7 +1084,18 @@ angular.module('openshiftConsole')
       return lastFinishTime;
     };
   })
+  // gets the first status condition.
   .filter('statusCondition', function() {
```

Comment:
No I disagree with this behavior change.  You may at any point have multiple status conditions, you need to actually ask for the one you want.